shaders/MicrobeBackground.gdshader

```diff
@@ -1,64 +1,77 @@
 shader_type spatial;
 render_mode unshaded;
 
-// this is 1 by default to keep the dark BGs looking nice
+// Default light level for dark backgrounds
 uniform float lightLevel = 1.0f;
 
 uniform vec2 repeats = vec2(1.0f, 1.0f);
-uniform sampler2D layer0 : source_color;
-uniform sampler2D layer1 : source_color;
-uniform sampler2D layer2 : source_color;
-uniform sampler2D layer3 : source_color;
+uniform sampler2D layer0 : source_color; 
+uniform sampler2D layer1 : source_color; 
+uniform sampler2D layer2 : source_color; 
+uniform sampler2D layer3 : source_color; 
 
-const vec2 speed0 = vec2(3300.0f);
-const vec2 speed1 = vec2(2550.0f);
-const vec2 speed2 = vec2(1800.0f);
-const vec2 speed3 = vec2(1050.0f);
+// the noise textures, distortion amount, and distortion speed for the distortion shader
+uniform sampler2D noise_tex_1; 
+uniform sampler2D noise_tex_2; 
+uniform float distortion_factor; 
+uniform float distortion_speed; 
 
-varying vec2 UV3;
-varying vec2 UV4;
+const vec2 speed0 = vec2(3300.0f); 
+const vec2 speed1 = vec2(2550.0f); 
+const vec2 speed2 = vec2(1800.0f); 
+const vec2 speed3 = vec2(1050.0f); 
 
-vec3 LightInfluence(float amount)
-{
-    if (amount > 0.5f)
-    {
-        // Day
+varying vec2 UV3; 
+varying vec2 UV4; 
+
+vec3 LightInfluence(float amount) { 
+    if (amount > 0.5f) {
         return mix(vec3(0.75f, 0.5f, 0.5f), vec3(1.0f, 1.0f, 1.0f), 2.0f * amount - 1.0f);
-    }
-    else if (amount > 0.25f)
-    {
-        // Dawn and Dusk
+    } else if (amount > 0.25f) {
         return mix(vec3(0.25f, 0.25f, 0.25f), vec3(0.75f, 0.5f, 0.5f), 4.0f * amount - 1.0f);
-    }
-    else
-    {
-        // Night
+    } else {
         return mix(vec3(0.052f, 0.05f, 0.17f), vec3(0.25f, 0.25f, 0.25f), 4.0f * amount);
     }
 }
 
-void vertex(){
+void vertex() { 
```

Comment:
Not doing these formatting changes (so undoing these) would make the diff a lot clearer, and avoiding reordering the speed variables etc. at the start of the file would help a lot with seeing what is going on.